src/Microsoft.AspNetCore.Razor.Language/AllowedChildTagDescriptor.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.AspNetCore.Razor.Language
 {
-    [DebuggerDisplay("{DisplayName,nq}")]
+    [DebuggerDisplay("{" + nameof(DisplayName) + ",nq}")]
```

Comment:
It **looked** better before. But as @pranavkm [said](https://github.com/aspnet/Razor/issues/1580#issue-245158823),  this prevents bugs like these from happening again.